template_rpm/09_cleanup.sh

```diff
@@ -49,4 +49,9 @@ fi
 
 truncate --no-create --size=0 "${INSTALL_DIR}"/var/log/dnf.*
 
+if [[ "$DIST_NAME" = 'fedora' ]] && [[ "$DIST_VER" -ge 37 ]] &&
+    rpm --root "$INSTALL_DIR" -qv qubes-core-agent-selinux; then
+    echo 'selinux=1' > /template.conf
```

Comment:
I'm pretty sure `/template.conf` is not correct here. See https://github.com/QubesOS/qubes-linux-template-builder/blob/main/qubeize_image#L117-L132 where it's looked for. Using `./template.conf` would work, as the "cleanup" script is called after that selection, but it isn't particularly nice.
The current template builder code covers only "static" template.conf (based on distribution version/flavor), not dynamically created. So, maybe add another case there that looks for example into `$(dirname "$IMG")` (or a new variable with this value)? And then generate template.conf in 04_install_qubes.sh.